apps/tehwolfde/src/app/i18n/language-switcher.component.scss

```
@@ -31,24 +33,48 @@
 }
 
-// The code is the only text in the toolbar that Material colours itself, so
+// The menu entries are styled in styles.scss, not here: mat-menu renders into
+// the CDK overlay at the document root, outside this component's DOM, so no
+// scoped rule reaches them.
+
+// The name is the only text in the toolbar that Material colours itself, so
 // without this it rendered in the default button foreground while the nav links
 // and icons beside it use the site link colour.
-.language-code {
+//
```

Stylelint (17.14.1)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)